test/fixtures/data/subscriptions/ropsten/dataFeeds/0x33ced632274973f86303f003416dfcb0d0a59aefe7a0f3fef5c42bb890383846-0xa55026ee522feb3c80cfccdd880865aeb9475a4a7675c036db89e4f6bc7c5a10.json

```diff
@@ -0,0 +1,12 @@
+{
+  "paymentTxHash": "0xa55026ee522feb3c80cfccdd880865aeb9475a4a7675c036db89e4f6bc7c5a10",
+  "resourceId": "0x33ced632274973f86303f003416dfcb0d0a59aefe7a0f3fef5c42bb890383846",
+  "claimaintAddress": "0x1a2633190693307d47145098fFd1d4669D3aE9eE",
+  "beneficiaryAddress": "0x25B246C3bA7B7353e286859FaE8913600b96B719",
+  "whitelistAddress": "0x25B246C3bA7B7353e286859FaE8913600b96B719",
+  "coverageAmount": "10000",
+  "startDate": 1653048763,
+  "endDate": 1653038763,
+  "ipfsPolicyHash": "QmTtDqWzp179ujTXU7pd2PodLNjpdpQQCXhkiQYi6wZvKd",
+  "ipfsServicePolicyHash": "QmRBQB6YaDyidP37UdDnjFY6vQuiBrcqdyoW1CuDgwxkD6"
+}
```

Comment:
Isn't this missing either `dapiName` or `dataFeedId`?

Reply:
You're correct, thank you!
Weird that the tests pass, maybe we're missing validation in the tests. I'll check, thanks again!

Reply:
Yes, we don't run validation on mock data in tests. I'll add this. Thanks again @acenolaza really good find.